torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -650,6 +655,13 @@ RegisterOperators reg(
            return 0;
          },
          aliasAnalysisFromSchema()),
+     Operator(
+         "aten::set_grad_enabled(bool mode) -> ()",
```

Comment:
This needs to be marked as conservative. Otherwise the compiler can move compilation above/below this function during optimizations.